hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -470,17 +470,17 @@ private long calculateMaxSteps(BalancerClusterState cluster) {
     updateStochasticCosts(tableName, curOverallCost, curFunctionCosts);
     if (initCost > currentCost) {
       List<RegionPlan> plans = createRegionPlans(cluster);
-      LOG.info("Finished computing new load balance plan. Computation took {}" +
-        " to try {} different iterations.  Found a solution that moves " +
-        "{} regions; Going from a computed cost of {}" +
-        " to a new cost of {}", java.time.Duration.ofMillis(endTime - startTime),
-        step, plans.size(), initCost, currentCost);
+      LOG.info("Finished computing new load balance plan. Computation took {} ms" +
+          " to try {} different iterations.  Found a solution that moves " +
+          "{} regions; Going from a computed cost of {}" +
+          " to a new cost of {}. Move plan to be submitted to master to execute",
```

Comment:
Rather than 'move plan to be submitted...' just say 'Executing plan'.

Its called 'cost' but is that the right term? Is it our assessment of 'balance'?

Reply:
It's not accurate to say we're executing the plan at this point though.

What ware we trying to convey with this addition? From the issue description I think it's that the act of selecting a plan for the cluster does not mean we've executed that plan. is that correct?

Reply:
Yes. @busbey after submitting the moves, balancer will rest for another time period before it resumes. If the moves are complete, it will look again to see if we need to move more. otherwise, it will skip.

Reply:
This addressed?